test/fixtures/flow/type-imports/invalid-import-type-shorthand/options.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You are right, babylon throws at the first error, and never parses the second line. I extracted that in a second test.